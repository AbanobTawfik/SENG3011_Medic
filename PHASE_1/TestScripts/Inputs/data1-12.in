#!/bin/sh
# an event where only the year is known

start_date=2010-12-23T00:00:00
end_date=2011-02-19T23:59:59
