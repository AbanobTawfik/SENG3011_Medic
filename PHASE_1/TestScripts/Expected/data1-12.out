{
    "meta": {
        "team_name": "Medics",
        "accessed_time": "REMOVED",
        "data_source": {
            "name": "CDC",
            "url": "https://www.cdc.gov/outbreaks/"
        }
    },
    "articles": [
        {
            "url": "url7",
            "reports": [
                {
                    "diseases": [],
                    "syndromes": [],
                    "event_date": "2011-xx-xx xx:xx:xx to 2012-xx-xx xx:xx:xx",
                    "locations": []
                }
            ],
            "headline": "Headline of url7",
            "main_text": "This is the main text for url7.",
            "date_of_publication": "2013-01-23 12:00:00"
        }
    ]
}
